### PR TITLE
fix: disable badge commit for now

### PR DESCRIPTION
**Pull request by RMCampos**
# Description

- Check-style fixes (already merged to main, due to unprotected branch, now fixed)
- Disable JaCoCo coverage badge commit when changed, for now, because it's undoing commit changes.

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)

# How Has This Been Tested?

- Tested with compilation
- Tested running locally

## Checklist

- [x] I have read the [CONTRIBUTING](CONTRIBUTING.md) doc
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have already been accepted and merged

### CODE REVIEW

File: src/test/java/ca/bc/gov/backendstartapi/endpoint/CheckEndpointIT.java [hunk 1]
```diff
@@ -17,21 +17,21 @@
 @WebFluxTest(CheckEndpoint.class)
 class CheckEndpointIT {
```
Comment (annibalsilva) [inline src/test/java/ca/bc/gov/backendstartapi/endpoint/CheckEndpointIT.java:18]: The suffix *IT* doesn't follow the camel case rules from the style guide (see the third step of https://google.github.io/styleguide/javaguide.html#s5.3-camel-case).

On another note, the guide says that test classes end with the *Test* suffix, although the style checker didn't raise any issues in this point.

File: src/test/java/ca/bc/gov/backendstartapi/dto/ExampleDtoTest.java [hunk 2]
```diff
@@ -5,57 +5,59 @@
 
 public class ExampleDtoTest {
```
Comment (annibalsilva) [inline src/test/java/ca/bc/gov/backendstartapi/dto/ExampleDtoTest.java:6]: The style guide says that every public class must have its Javadoc (see https://google.github.io/styleguide/javaguide.html#s7.3-javadoc-where-required).

**Review by annibalsilva — CHANGES_REQUESTED** [review]
In addition to CI undoing the ordering of the imports, two style violations were found.

File: src/test/java/ca/bc/gov/backendstartapi/dto/ExampleDtoTest.java (hunk 2 above)
Comment (RMCampos) [reply src/test/java/ca/bc/gov/backendstartapi/dto/ExampleDtoTest.java:6]: Great point. Thanks.

File: src/test/java/ca/bc/gov/backendstartapi/endpoint/CheckEndpointIT.java (hunk 1 above)
Comment (RMCampos) [reply src/test/java/ca/bc/gov/backendstartapi/endpoint/CheckEndpointIT.java:18]: Nice. Fixed, thanks!

**Review by annibalsilva — APPROVED** [review]
We should just change the squashed commit message to address the only change being made, that is, the disabling commits about the coverage badge.

**Pull request merged**